packages/jellyfish-api-core/__tests__/category/account/accountToAccountHistory.test.ts

```diff
@@ -0,0 +1,64 @@
+
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload } from '../../../src/category/account'
+
+describe('listAccountHistory', () => {
```

Comment:
yup this works, but instead of having multiple files, you should scope them into describe blocks as one file instead.